thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/client/ThirdEyeCacheRegistry.java

```diff
@@ -167,10 +169,17 @@ private static void initCommonCacheLoaders() {
 
     ThirdEyeCacheRegistry cacheRegistry = ThirdEyeCacheRegistry.getInstance();
 
+    RemovalListener<PinotQuery, ResultSetGroup> listener = new RemovalListener<PinotQuery, ResultSetGroup>() {
+      
+      @Override
+      public void onRemoval(RemovalNotification<PinotQuery, ResultSetGroup> notification) {
+        LOGGER.info("Expired {}", notification.getKey().getPql());
+      }
+    };
     // ResultSetGroup Cache
     // TODO: have a limit on key size and maximum weight
     LoadingCache<PinotQuery, ResultSetGroup> resultSetGroupCache =
-        CacheBuilder.newBuilder().expireAfterAccess(1, TimeUnit.HOURS)
+        CacheBuilder.newBuilder().removalListener(listener).expireAfterAccess(1, TimeUnit.HOURS)
```

Comment:
I thought it will be a good tool to understand how often items get evicted and may be come up with a better eviction strategy.